src/Admin/AbstractAdmin.php

```diff
@@ -1920,9 +1928,31 @@ public function getChild($code)
         return $this->hasChild($code) ? $this->children[$code] : null;
     }
 
-    public function setParent(AdminInterface $parent)
+    public function setParent(AdminInterface $parent/*, string $parentAssociationMapping*/)
```

Comment:
Please add PHPDoc, so we can get autocompletion

Reply:
I tried, but I have no idea how to add it because it gets removed by the php-cs-fixer.

Reply:
```suggestion
    /**
     * @param AdminInterface $parent                    Parent admin class.
     * @param string         $parentAssociationMapping  Set $parent to this filed. 
     */
    public function setParent(AdminInterface $parent/*, string $parentAssociationMapping*/)
```
Add some comments then php-cs-fixer will not remove it. This change will not autocomplite second parameter. (In PhpStorm).

Reply:
Thanks @wbloszyk, but with the above code, the `@param string         $parentAssociationMapping  Set $parent to this filed.` part gets removed by the php-cs-fixer. 

I could add a normal comment, but the point of this was to use PHPDoc to get autocompletion, [looks like in the past it existed an `optional` argument](https://manual.phpdoc.org/HTMLframesConverter/default/), but I haven't found a way now.

Reply:
I check it only for $parent param. I think we can't add autocomplite without API change or change php-cs-fixer config what we shouldn't do.